django/http/multipartparser.py

```diff
@@ -580,7 +580,7 @@ def exhaust(stream_or_iterable):
     if iterator is None:
         raise MultiPartParserError('multipartparser.exhaust() was passed a non-iterable or stream parameter')
 
-    for __ in iterator:
+    for _ in iterator:
```

Comment:
I would suggest not using `_` as a variable name as that may conflict with gettext imports.